main.js

```diff
@@ -0,0 +1,106 @@
+// mobile menu
+/* eslint-disable linebreak-style */
+const menu = document.querySelector('.mobile-menu-container');
+const menuOpenButton = document.querySelector('.burger-icon');
+const menuCloseButton = document.querySelector('.menu-close-button');
+
+menuOpenButton.addEventListener('click', () => {
+  menu.style.display = 'flex';
+  menuCloseButton.style.display = 'flex';
+});
+
+menuCloseButton.addEventListener('click', () => {
+  menu.style.display = 'none';
+  menuCloseButton.style.display = 'flex';
+});
+
+// document.querySelector('.mobile-menu-close').addEventListener('click', (e) => {
+// document.querySelector('.mobile-menu-container').setAttribute('style', 'visibility: hidden');
+// e.preventDefault();
+// });
+
+// document.querySelector('.burger-icon-single').addEventListener('click', () => {
+// document.querySelector('.mobile-menu-container').setAttribute('style', 'visibility:hidden');
+// });
+
+// document.querySelector('.burger-icon-single').addEventListener('click', () => {
+// document.querySelector('.mobile-menu-container').setAttribute('style', 'visibility:hidden');
+// });
+
+// document.querySelector('.burger-icon-single').addEventListener('click', () => {
+// document.querySelector('.mobile-menu-container').setAttribute('style', 'visibility:hidden');
+// });
+
+// Feautre Speaker
```

Comment:
- [ ] Kindly remove all commented lines to make your project more clean

Reply:
if i remove the commented line . its going to give me error
so how i fix this kind of issue.?